lib/web_ui/test/canvaskit/text_test.dart

```diff
@@ -94,6 +94,42 @@ void testMain() {
       expect(await matchImage(tabImage, tofuImage), isFalse);
     });
 
+    test('kTextHeightNone unsets the height multiplier', () {
```

Comment:
Can you move these tests into one of the text tests under `flutter/lib/web_ui/test/ui` so that it runs on all the renderers instead of just CanvasKit? As-is, these unit tests are not testing skwasm.